.github/workflows/ci.yml

```diff
@@ -32,45 +37,107 @@ jobs:
           username: ${{ secrets.FM_DOCKERHUB_TOKEN_USER }}
           password: ${{ secrets.FM_DOCKERHUB_TOKEN }}
 
-      - name: Prepare lite-build image
+      - name: Rebuild and push build image
```

Comment:
probably should say "if needed" like the mariadb job. Same for run base image on line 54.